assets/css/stripe-link.css

```diff
@@ -10,7 +10,7 @@
 	height: 40px;
 	background: no-repeat
 		url( '../../client/payment-method-icons/link/icon.svg' );
-	background-color: none;
+	background-color: transparent !important;
```

Comment:
Out of curiosity, why `!important`?

Reply:
I think this is not taking effect, @dpaun1985. Having the background transparent, would make the background look white.

<img width="443" alt="image" src="https://user-images.githubusercontent.com/10233985/180038228-3306fe53-2a0f-4cfb-b042-d5396a6f1c2b.png">

Reply:
this was introduced by @mordeth . From what I remember, he told me that this was introduced because some of the themes are overwriting the background. 
So, here is not taking effect, but for other themes it will.

Reply:
It was added because some themes like Twenty Twenty-One have `background-color` with higher specificity and overwrites Link button styles. I am aware `!important` shouldn't be used, but I think in that case it is worth keeping it.